bedrock/settings/base.py

```diff
@@ -15,7 +15,9 @@
 from pathlib import Path
 from urllib.parse import urlparse
 
-from django.conf.locale import LANG_INFO  # we patch this in bedrock.base.apps.BaseAppConfig  # noqa: F401
+from django.conf.locale import (
+    LANG_INFO,
+)  # we patch this in bedrock.base.apps.BaseAppConfig  # noqa: F401
```

Comment:
So the linter fails here, because it seems the IDE unintentionally makes formatting changes in .py files behind your back — with the unfortunate coincidence here that when reformatted, the `noqa` ignore rule ends up on a different line, thus drawing the anger of the formatters to the line it was supposed to protect from their nosy checks;)

(Worse, the same change ended up in the sibling PR too, making them conflicting with each other, asking for headaches when merging…)

Also check the rest of the file, there seem to be formatting changes unrelated to this PR, similar to what I've noted in one of the previous PRs…
 
(The other linter fail here is in Fluent group-level comments, the log will tell you to add an extra line after the heading… that should silence it.)